CMake CUDA SDK seem broken - Couldn't find CUDA library root.

I have build an sdk with follwoing content

```
TOOLCHAIN_HOST_TASK += "nativesdk-cmake \
    nativesdk-packagegroup-cuda-sdk-host \
    nativesdk-cuda-environment \
 "

TOOLCHAIN_TARGET_TASK += "
    cuda-compiler-dev \
    cuda-cudart-dev \
    cuda-toolkit-dev \
"
```

I have a simple CMakeLists.txt  with

```
cmake_minimum_required(VERSION 3.17)
project(test_cuda LANGUAGES CXX CUDA)
find_package(CUDA REQUIRED)
```

*scarthgap-l4t-r35.x branch work fine and I get: *

```
-- The CXX compiler identification is GNU 13.2.0
_CUDA_NVCC_EXECUTABLE
/opt/tegra-grimme/4.0.6+snapshot/sysroots/x86_64-tdsdk-linux/usr/local/cuda-11.4/bin/nvcc
-- The CUDA compiler identification is NVIDIA 11.4.315
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /opt/tegra-grimme/4.0.6+snapshot/sysroots/x86_64-tdsdk-linux/usr/bin/aarch64-oe4t-linux/aarch64-oe4t-linux-g++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Detecting CUDA compiler ABI info
-- Detecting CUDA compiler ABI info - done
-- Check for working CUDA compiler: /opt/tegra-grimme/4.0.6+snapshot/sysroots/x86_64-tdsdk-linux/usr/local/cuda-11.4/bin/nvcc - skipped
-- Detecting CUDA compile features
-- Detecting CUDA compile features - done
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
-- Found Threads: TRUE  
-- Found CUDA: /opt/tegra-grimme/4.0.6+snapshot/sysroots/x86_64-tdsdk-linux/usr/local/cuda-11.4 (found version "11.4") 
-- Configuring done (7.1s)
-- Generating done (0.0s)
-- Build files have been written to: /egbe/test_cuda/build
```

but with scarthgap cmake fails and I get: 

```
-- The CXX compiler identification is GNU 13.2.0
CMake Error at /opt/tegra-grimme/4.0.6+snapshot/sysroots/x86_64-tdsdk-linux/usr/share/cmake-3.28/Modules/Internal/CMakeCUDAFindToolkit.cmake:175 (message):
  Couldn't find CUDA library root.
Call Stack (most recent call first):
  /opt/tegra-grimme/4.0.6+snapshot/sysroots/x86_64-tdsdk-linux/usr/share/cmake-3.28/Modules/CMakeDetermineCUDACompiler.cmake:89 (cmake_cuda_find_toolkit)
  CMakeLists.txt:3 (project)
-- Configuring incomplete, errors occurred!
```

--- a/recipes-devtools/cuda/cuda-cudart_12.6.68-1.bb
+++ b/recipes-devtools/cuda/cuda-cudart_12.6.68-1.bb
@@ -28,5 +28,5 @@ FILES:${PN}-dev += "${prefix}/local/cuda-${CUDA_VERSION}/${baselib}/*.a \
                     ${@' ${prefix}/local/cuda-${CUDA_VERSION}/lib64' if d.getVar('baselib') != 'lib64' and d.getVar('SITEINFO_BITS') == '64' else ''}"
 FILES:${PN}-staticdev = ""
 INSANE_SKIP:${PN}-dev += "staticdev"
-RDEPENDS:${PN}-dev:append:class-target = " cuda-nvcc-headers cuda-cccl cuda-target-environment cuda-crt"
+RDEPENDS:${PN}-dev:append:class-target = " cuda-nvcc-headers cuda-cccl cuda-target-environment cuda-crt-dev"
 BBCLASSEXTEND = "native nativesdk"

--- a/recipes-devtools/cuda/cuda-nvcc_12.6.68-1.bb
+++ b/recipes-devtools/cuda/cuda-nvcc_12.6.68-1.bb
@@ -17,6 +17,6 @@ FILES:${PN}-dev = ""
 INSANE_SKIP:${PN} += "dev-so dev-deps"
 RDEPENDS:${PN} = "${BPN}-headers"
 RDEPENDS:${PN}:append:class-target = " cuda-nvvm-dev cuda-crt-dev"
-RDEPENDS:${PN}:append:class-nativesdk = " nativesdk-cuda-environment"
+RDEPENDS:${PN}:append:class-nativesdk = " nativesdk-cuda-environment nativesdk-cuda-nvvm"
 
 BBCLASSEXTEND = "native nativesdk"

